handler/src/main/java/io/netty/handler/pcap/PcapWriteHandler.java

```diff
@@ -517,7 +532,21 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
             logger.debug("Sent Fake TCP RST to close connection");
         }
 
-        this.pCapWriter.close();
+        close();
         ctx.fireExceptionCaught(cause);
     }
+
+    /**
+     * <p> Close {@code PcapWriter} and {@link OutputStream}. </p>
+     * <p> Note: Calling this method does not close {@link PcapWriteHandler}.
+     * Only Pcap Writes are closed. </p>
```

Comment:
The close method closes `OutputStream` and `PcapWriter` so that, Pcap writes are stopped. The handler is not closed because we want data to still travel across.